src/dmd/backend/elfobj.d

```diff
@@ -299,6 +300,9 @@ int         MAP_SEG2TYP(int seg)    { return MAP_SEG2SEC(seg).sh_flags & SHF_EXE
 public seg_data **SegData;
 public int seg_length;          // used length of SegData[]
```

Comment:
These variables seem to be redundant to SegDataR now?

Reply:
They are still used in the file and probably will be removed by following pull requests. Might be useful to move all SegData refactoring to separate branch

Reply:
Right. Messing with memory allocation is tricky and I want to minimize diffs while doing that.

Reply:
Fair enough